akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingGetStatsSpec.scala

```diff
@@ -109,17 +109,18 @@ abstract class ClusterShardingGetStatsSpec extends MultiNodeSpec(ClusterSharding
       join(second)
       join(third)
 
-      // make sure all nodes has joined
-      awaitAssert {
-        Cluster(system).sendCurrentClusterState(testActor)
-        expectMsgType[CurrentClusterState].members.size === 3
+      // make sure all nodes are up
+      within(10.seconds) {
+        awaitAssert {
+          Cluster(system).state.members.count(_.status == MemberStatus.Up) === 4
```

Comment:
I don't want it to pass here until there are 4 nodes up and seen, failing the assertion throws an exception, so that will make it try again for up to 10 seconds? (Could be another case of not knowing the testkit well enough)